vignettes/reuse.Rmd

```diff
@@ -35,6 +35,53 @@ You can document multiple functions in the same file by using either `@rdname` o
 It's a technique best used with care: documenting too many functions in one place leads to confusion.
 Use it when all functions have the same (or very similar) arguments.
 
+### `@rdname`
+
+Use `@rdname <destination>` to include multiple functions in the same page.
+The functions will be collected and share the same `@title`, `@description`, `@examples`, etc.
+
+There are two basic ways to use it.
+You can create a dummy documentation block and then add the functions to it.
+This typically works best when you have a family of related functions and you want to link to that family from other functions (i.e. `trig` in the examples below).
+
+```{r}
+#' Trigonometric approximations
+#' @param x Input, in radians.
+#' @name trig
+NULL
+
+#' @rdname trig
+#' @export
+sin_ish <- function(x) x - x^3 / 6
+
+#' @rdname trig
+#' @export
+cos_ish <- function(x) 1 - x^2 / 2
+
+#' @rdname trig
+#' @export
+tan_ish <- function(x) x + x^3 / 3
+```
+
+Alternatively, you can add docs to an existing function.
+This tends to work better if you have a "primary" function with some variants or some helpers.
```

Comment:
I'm not sure we're _quite_ there yet, now we have in `@rdname`

> This tends to work better if you have a "primary" function with some variants or some helpers.

Vs in `@describeIn`

> You can document several functions which perform slight variations of a task or which use different defaults

Do we need to distinguish these two "variant"/'variation" use cases further? Or are they \~somewhat\~ interchangeable?

Reply:
TBH I've mostly forgotten what `@describeIn` does and why you might use it. I think the main difference is that it attempts to build the single description for you by creating a bulleted list. But I'm not sure this is now possible to do in a compelling way, and I think you're generally better off using `@rdname` with a custom `@description`.

Reply:
That seems reasonable enough to me. OTOH, as someone looking at a lot of other peoples' code, it's something I come across fairly often -- it would be nice to point to a canonical reference during code review urging one or the other in each case.

I see 1000s of usages on CRAN:

https://github.com/search?q=lang%3AR+%2F%23%27%5Cs%40describeIn%2F+org%3Acran&type=code

Here's the only file with usage in tidyverse:

https://github.com/tidyverse/reprex/blob/b904f73989583e66cb02fd0199df6cede5be1c3f/R/reprex-undo.R#L22

Vs. 10 packages using it from r-lib:

https://github.com/search?q=lang%3AR+%2F%23%27%5Cs%40describeIn%2F+org%3Ar-lib&type=code

Reply:
Ok, I've made it more clear that I don't recommend it, and I'll fix the reprex usage momentarily.

Reply:
https://github.com/tidyverse/reprex/pull/452